onboarding.md

```diff
@@ -0,0 +1,43 @@
+# Onboarding
+
+This document covers steps needed to onboard new Steering Committee
+members and off-board emeritus members.
+
+- [ ] All new members must complete the [Inclusive Leadership Training].
+
+- [ ] Update the following files in the respective repos:
+  - [ ] [kubernetes/steering]: `OWNERS_ALIASES` and `README.md`
+  - [ ] [kubernetes/community]: `sigs.yaml` and auto-generated content
+  - [ ] [kubernetes/org]: `OWNERS_ALIASES` and the `steering-committee`
+    GitHub team
+  - [ ] [kubernetes/funding]: `OWNERS_ALIASES`
+  - [ ] [kubernetes/kubernetes-template-project]: `OWNERS_ALIASES`
+
+- [ ] Add new members with Owner access to the `steering-private`
+  mailing list and remove emeritus members.
+
+- [ ] Grant new members Owner access to the public steering mailing
+  list and revoke access from emeritus members.
+
+- [ ] Add new members to the `steering-private` slack channel and
+  remove emeritus members.
+
+- [ ] Add new members to the CNCF Service Desk and remove emeritus
+  members.
+
+- [ ] Add new members to the [cncf-kubernetes-maintainers] mailing list.
+
+- [ ] If a GSuite top-level account is owned by an emeritus member,
```

Comment:
@dims @parispittman should any more details added here? Is the process straight-forward or would it benefit from additional details here?

Reply:
we can link to this - https://github.com/kubernetes/steering#top-level-accounts here so we know where to look quickly for the names of these folks. (i also need to replace jbedas name with mine here hehe)